src/schema/v2/user.ts

```diff
@@ -62,11 +77,34 @@ export const UserSaleProfileField: GraphQLFieldConfig<any, ResolverContext> = {
   },
 }
 
+export const UserAdminNotesField: GraphQLFieldConfig<any, ResolverContext> = {
+  description: "The admin notes associated with the user",
+  type: new GraphQLList(UserAdminNoteType),
+  resolve: ({ id }, {}, { userAdminNotesLoader }) => {
+    if (!userAdminNotesLoader) {
+      throw new Error(
+        "You need to be signed in as an admin to perform this action"
+      )
+    }
+
+    return userAdminNotesLoader(id)
+      .then((result) => {
+        return result
+      })
+      .catch((err) => {
```

Comment:
I would avoid `catch` unless needed which I don't think it is here? You'd want to propagate any errors up (authentication or authorization related, 'not found', etc.) as normal , so just returning the loader.